atd-etl/cris_import/cris_import.py

```diff
@@ -107,9 +115,45 @@ def main():
             typed_token = align_db_typing(trimmed_token)
             align_records_token = align_records(typed_token)
             clean_up_import_schema(align_records_token)
+        mark_extract_as_imported(archive_data[0], database_location)
     if not local_mode:  # We're using a locally provided zip file, so skip these steps
-        remove_archives_from_sftp_endpoint(zip_location)
-        upload_csv_files_to_s3(archive)
+        upload_sqlite_to_s3(database_location)
```

Comment:
Here is where we put our modified SQLite DB back on S3. This system includes no provisions for locking this file, and as a result, there can never be two CRIS import scripts running at once. This is a given in our nominal operation, but it is worth mentioning.